.eslintrc.cjs

```diff
@@ -1,15 +1,14 @@
 module.exports = {
 	root: true,
-	extends: ['@sveltejs', 'prettier'],
+	extends: ['@sveltejs'],
 	settings: {
 		'import/core-modules': [
 			'svelte',
 			'svelte/internal',
 			'svelte/store',
 			'svelte/easing',
 			'estree'
-		],
-		'svelte3/compiler': require('./compiler')
```

Comment:
What was this doing?

Reply:
It was for the old svelte eslint plugin. I migrated from the eslint-plugin-svelte3 to eslint-plugin-svelte3, so it's no longer required